roboprop_client/views.py

```diff
@@ -97,6 +100,64 @@ def _get_model_details(result):
     }
 
 
+def __remove_outliers_and_sort(items):
+    # Remove single occurences as is most likely an outlier
+    items = [item for item in items if items.count(item) > 1]
+    # Sort by most occurences
+    sorted(items, key=lambda x: items.count(x), reverse=True)
+    # Remove duplicates
+    items = list(set(items))
+    return items
+
+
+def __detect_thumbnail_details(thumbnail):
+    client = boto3.client("rekognition")
+
+    response = client.detect_labels(
+        Image={"Bytes": base64.b64decode(thumbnail)},
+        Features=["GENERAL_LABELS", "IMAGE_PROPERTIES"],
+        MinConfidence=90,
```

Comment:
Out of Curiosity, did you try tweaking this?

Reply:
Indeed yes! Confidence in my tests tends to be relatively accurate at around the 93 area, so 90 seems to be a sweetspot of removing nonsense, but avoiding possible missed (and accurate) descriptions

Reply:
Did you only test on the Cessna or also on other models (like kitchen related ones)

Reply:
I had tested on various other models. (cafe tables, sink, coffee maker, fridge).

There are some outliers, for example with Fridge, if min confidence is 60, it does guess that its a refrigerator. But it also guesses a bunch of other nonsense like "city" and "mailbox", to where, in my mind, it seems a bit "plucking at straws". So I think a high confidence, with things being missed out perhaps, but a more concise list, with the option to add your own is perhaps better?

Reply:
Thank you for the details. I agree.
It might be nice to have a small comment regarding this value in the code